lib/octokit/client/reviews.rb

```diff
@@ -160,6 +160,22 @@ def request_pull_request_review(repo, id, reviewers, options = {})
         options = options.merge(reviewers: reviewers)
         post "#{Repository.path repo}/pulls/#{id}/requested_reviewers", options
       end
+
+      # Delete a review request
+      #
+      # @param repo [Integer, String, Hash, Repository] A GitHub repository
+      # @param id [Integer] The id of the pull request
+      # @param reviewers [Array<String>] An array of user logins that will be removed
```

Comment:
The payload that the endpoint takes is an object not an array:

```
{
  "reviewers": [
    "octocat",
    "hubot",
    "other_user"
  ],
  "team_reviewers": [
    "justice-league"
  ]
}
```

We need to be able to handle deleting both user review requests and team review requests.